doc/source/examples/index.rst

```diff
@@ -18,17 +18,18 @@ Here are a series of examples using MAPDL with ``ansys-mapdl-core``.
 
 .. === TECHNOLOGY SHOWCASE EXAMPLES ===
 
-.. toctree::
-    :hidden:
-    :maxdepth: 3
-
-    technology_showcase_examples/techdemo-1/ex_0-tecbrakesqueal
-    technology_showcase_examples/techdemo-20/20-example-technology-showcase-dynamic-simulation-PCB
-    technology_showcase_examples/techdemo-28/ex_28-tecfricstir
-
-.. include:: technology_showcase_examples/index.rst
-   :start-line: 2
-
+The PyMAPDL technology showcase examples are more complex test cases than
+the ones presented in the <gallery_examples_>. They demonstrate how to use
+PyMAPDL to effectively and accurately solve interdisciplinary problems from
+a variety of industries and engineering fields.
+
+The examples initially come from the `APDL Technology Showcase Manual
+<_tech_demo_intro_>`_ and are modeled with MAPDL files. Some of them have been
+reproduced in Python files using PyMAPDL with the ``ansys-mapdl-core``
+library.
+
+The documentation to the exposed examples can be found in the
+`pymapdl-techdemos <_tech_demos_repo_>`_ repository.
```

Comment:
Watch out! One thing is the repository where the techdemos are hosted (https://github.com/pyansys/pymapdl-techdemos) another thing is where the documentation is hosted (https://techdemos.mapdl.docs.pyansys.com/)

While providing a link to the repository is needed, the most important part is linking to the actual documentation.

The link you defined should point to the repository, and you should create a new link called ``tech_demos_docs`` that points to the documentation.

Reply:
Thanks, I'll change it right now